inspirehep/modules/workflows/models.py

```diff
@@ -50,3 +58,16 @@ def save(self):
         """Save object to persistent storage."""
         with db.session.begin_nested():
             db.session.add(self)
+
+
+class WorkflowsPendingRecord(db.Model):
+
+    __tablename__ = "workflows_pending_record"
+
+    workflow_id = db.Column(db.Integer, primary_key=True, nullable=False)
+    record_id = db.Column(db.Integer, nullable=False)
+
+    def save(self):
```

Comment:
Just copied it from above, not sure why it's needed. Maybe it's a way to avoid having to run the db.session.add explicitly?